docs/advanced-ai/intro-tutorial.md

```diff
@@ -1,217 +1,127 @@
+
+## 4. Configure the node
+  
+- AI agents require a chat model (e.g., a large language model like GPT) to function.
+- Add a chat model by clicking the plus (+) button and connecting to a model (e.g., OpenAI GPT).
+
+## 5. Add credentials (if needed)
+
+- explain credentials
+
+??? explanation "Keeping your credentials safe"
+    Some text with a more detailed explanation of this step
+
+
+## 6. Test the node:
+
+- Use the chat trigger to test the workflow by sending a sample message (e.g., “Hello”).
+- Check the logs to see:
+    - Input Data: Data received from the trigger (e.g., session ID, chat message).
+    - Output Data: The response generated by the AI agent (e.g., “Hello, how can I assist you today?”).
+
+## 7. Changing the prompt:
+
+- changing the prompt settings
+- demonstrate the changes
+
+## 8. Persistence:
+    
+- demonstrate lack of persistence
+- add memory
+
+
+## 9. saving the workflow
+
+## Congratulations!
```

Comment:
It's always good to remind people of what they did, it's a reinforced learning thing that really works!

Reply:
Yeah, that's definitely a good touch for tutorials. Good call out.